src/libxrpl/protocol/LedgerFormats.cpp

```diff
@@ -365,6 +365,18 @@ LedgerFormats::LedgerFormats()
         },
         commonFields);
 
+    add(jss::PermissionedDomain,
+        ltPERMISSIONED_DOMAIN,
+        {
+            {sfOwner,               soeREQUIRED},
+            {sfSequence,            soeREQUIRED},
```

Comment:
Do we really need sfSequence here? Do you ever use it again? Maybe its used later for something?

Reply:
It's in https://github.com/XRPLF/XRPL-Standards/discussions/228 and https://github.com/XRPLF/XRPL-Standards/blob/cdffdea5e3d95171eee72b4bf99bff32e5488d04/XLS-0080d-permissioned-domains/README.md ( @mvadari ? )

Reply:
It's included here because it's used in the object ID hash. See https://github.com/XRPLF/rippled/issues/3941

Reply:
I guess thats new? Because its not used on NFTokenOffer, Escrow or PayChan. Personally I cant think of how it adds value. If you have the ledger entry then you have the `index`/ID. If you want the txn it was included in then you have PreviousTxnID. But if its something we really want then I will start including it in my proposals.